mathml/global_attributes.json

```diff
@@ -392,19 +417,25 @@
           "mdn_url": "https://developer.mozilla.org/docs/Web/MathML/Global_attributes/mathvariant",
           "spec_url": "https://w3c.github.io/mathml-core/#dfn-mathvariant",
           "support": {
-            "chrome": {
-              "version_added": "84",
-              "partial_implementation": true,
-              "notes": "The only value supported is 'normal'.",
-              "impl_url": "https://crrev.com/c/2161013",
-              "flags": [
-                {
-                  "type": "preference",
-                  "name": "#enable-experimental-web-platform-features",
-                  "value_to_set": "Enabled"
-                }
-              ]
-            },
+            "chrome": [
+              {
+                "version_added": "109",
+                "notes": "The only value supported is 'normal'."
```

Comment:
I guess `"partial_implementation": true,` still applies with 109?

Reply:
You are right. (there seem to be a consensus to remove the other mathvariant values from MathML Core, but that hasn't be done yet)